index.html

```diff
@@ -0,0 +1,448 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-app-capable" content="yes">
+    <title>Space Ripples Reveal Big Bang's Smoking Gun -The New York Times</title>
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <script>
+      function cl() {
+        var btn = document.getElementById('searchbar');
+        btn.innerHTML = '<input type="text">';
+      }
+      function hm() {
+        var hhm = document.getElementById('hamb');
+        var hmtx = document.getElementById('news');
+        var htx = document.getElementById('arts');
+        var content = document.getElementsByClassName('hammenu');
+        hhm === checked ? content.innerHTML = hmtx + '\n' + htx :  '';
+      }
+    </script>
+</head>
+<body lang="en">
+    <div class="topNav">
+        <nav>
+          <section class="hamburger">
+            <label for="hamb" onclick="hm()">&#9776;</label>
+            <input type="checkbox" id="hamb">
+          </section>
+          <section class="search">
+            <span id="searchbar"></span>
```

Comment:
This span with the `searchbar` id adds no value to your project. Please consider removing it.